tests/test_bot_slack.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from unittest.mock import patch
+
+import node_monitor.load_config as c
+from node_monitor.bot_slack import SlackBot
+
+
+@patch("slack_sdk.WebClient")
+def test_send_message(mock_web_client):
+    mock_client = mock_web_client.return_value
+
+    expected_channel = "#node-monitor"
+    expected_message = "Hello, Slack!"
+
+    slack_bot = SlackBot(c.TOKEN_SLACK)
+    slack_bot.send_message(expected_channel, expected_message)
+
+    mock_client.chat_postMessage.assert_called_once_with(
+        channel=expected_channel,
+        text=expected_message)
+
+
+@pytest.mark.live_slack
+def test_send_message_slack():
+    """Send a real test message to a Slack workspace"""
+    slack_bot = SlackBot(c.TOKEN_SLACK)
+    
+    slack_channel_name = "node-monitor-allusion"
+    message = "🔬 Hello from test_send_message_slack()"
```

Comment:
This will send test messages to the slack channel that we are using in production. 

I have a separate slack workspace that I use to send slack messages there. So instead, we should just send test messages to the following channel:
```suggestion
def test_send_message_slack():
    """Send a real test message to a Slack workspace"""
    slack_bot = SlackBot(c.TOKEN_SLACK)
    
    slack_channel_name = "node-monitor"
    message = "🔬 Hello from test_send_message_slack()"
```